Schema/V1/no.ks.fiks.arkiv.v1.innsyn.sok.xsd

```diff
@@ -190,17 +190,80 @@
     </xs:simpleType>
 
     <!-- The respons is strict, meaning that the result should return the exact type set here. E.g. when set to mappe it should only return mappe in resultList -->
-    <xs:simpleType name="respons">
+    <xs:complexType name="respons">
+        
+    </xs:complexType>
+
+    <xs:complexType name="mappeRespons">
+        <xs:complexContent>
+            <xs:extension base="respons">
+                <xs:sequence>
+                    <xs:element name="inkluder" type="inkluderIMappe"/>
+                </xs:sequence> 
+            </xs:extension>
+        </xs:complexContent>
+    </xs:complexType>
+
+    <xs:complexType name="saksmappeRespons">
+        <xs:complexContent>
+            <xs:extension base="respons">
+                <xs:sequence>
+                    <xs:element name="inkluder" type="inkluderISaksmappe"/>
+                </xs:sequence>
+            </xs:extension>
+        </xs:complexContent>
+    </xs:complexType>
+
+    <xs:complexType name="registreringRespons">
+        <xs:complexContent>
+            <xs:extension base="respons">
+                <xs:sequence>
+                    <xs:element name="inkluder" type="inkluderIRegistrering"/>
+                </xs:sequence>
+            </xs:extension>
+        </xs:complexContent>
+    </xs:complexType>
+
+    <xs:complexType name="journalpostRespons">
+        <xs:complexContent>
+            <xs:extension base="respons">
+                <xs:sequence>
+                    <xs:element name="inkluder" type="inkluderIJournalpost"/>
+                </xs:sequence>
+            </xs:extension>
+        </xs:complexContent>
+    </xs:complexType>
+
+    <xs:complexType name="dokumentbeskrivelseRespons">
+        <xs:complexContent>
+            <xs:extension base="respons">
+                <xs:sequence>
+                    <xs:element name="inkluder" type="inkluderIDokumentbeskrivelse"/>
+                </xs:sequence>
+            </xs:extension>
+        </xs:complexContent>
+    </xs:complexType>
+
+    <xs:simpleType name="inkluder">
```

Comment:
Denne utgår vel?

Reply:
Ja, jeg hadde den bare der for å se hva som var utgangspunktet